hash/sega_beena_cart.xml

```diff
@@ -13,21 +13,21 @@ license:CC0
+				<rom loadflag="load16_word_swap" name="S-100034-0041 - 1-nichi 10-pun de E ga Jouzu ni Kakeru Beena (Japan).bin" size="0x800000" crc="e823b0fd" sha1="ade6a02a516e87b21320def50f46d0414d35abac"/>
 			</dataarea>
 			<dataarea name="pages" size="0x2500000"> <!-- book pages -->
-				<rom name="0 - 0001.png" size="0x11b6a4c" crc="f8e2e06f" sha1="98d9b75d8324925cdc8996634c192f54b191ab3c"  />
-				<rom name="0 - 0002.png" size="0x183cb1c" crc="e88c8f6e" sha1="9f649a332d0a1be0f3566c63a85a21b1e5618e8b"  />
-				<rom name="0 - 0003.png" size="0x197c6c4" crc="30182f89" sha1="9cb9ccee2d21aaf7358ebfca21cb37b332cb5a98"  />
-				<rom name="0 - 0004.png" size="0x1867cca" crc="301d2785" sha1="302902938fe6013d5d3254f8738b07b9494a9856"  />
-				<rom name="0 - 0005.png" size="0x183b750" crc="d61c2830" sha1="fa92264b6f96d49c34c8b06b44469a5e430374f2"  />
-				<rom name="0 - 0006.png" size="0x178d006" crc="be606943" sha1="e28069d2608cb1bb13a3753ba520a0254112e86b"  />
-				<rom name="0 - 0007.png" size="0x191749e" crc="8bfd1ea8" sha1="1a9b78ac4238dfa437285e42f0bc7f8fd4a45e92"  />
-				<rom name="0 - 0008.png" size="0x1756da2" crc="720920cb" sha1="9291de9a4ecbe42ad3eb0fceb03edaf3824de3e9"  />
-				<rom name="0 - 0009.png" size="0x178b5cb" crc="c1c51104" sha1="a2ff91851ebae16649a06474408bea1a2fcb9aba"  />
-				<rom name="0 - 0010.png" size="0x19850b0" crc="520272eb" sha1="5d40da3e50581201c87f5460bbef1c328675a3cc"  />
-				<rom name="0 - 0011.png" size="0x19b8863" crc="77aa6918" sha1="b2095780c92fb55926e8e26759ebc803fe78d677"  />
-				<rom name="0 - 0012.png" size="0x14630ba" crc="d4a4f360" sha1="af125bddb75aba6ffca1a68dd6f226a15cbd7ce6"  />
+				<rom name="0 - 0001.png" size="0x11b6a4c" crc="f8e2e06f" sha1="98d9b75d8324925cdc8996634c192f54b191ab3c"/>
+				<rom name="0 - 0002.png" size="0x183cb1c" crc="e88c8f6e" sha1="9f649a332d0a1be0f3566c63a85a21b1e5618e8b"/>
+				<rom name="0 - 0003.png" size="0x197c6c4" crc="30182f89" sha1="9cb9ccee2d21aaf7358ebfca21cb37b332cb5a98"/>
+				<rom name="0 - 0004.png" size="0x1867cca" crc="301d2785" sha1="302902938fe6013d5d3254f8738b07b9494a9856"/>
+				<rom name="0 - 0005.png" size="0x183b750" crc="d61c2830" sha1="fa92264b6f96d49c34c8b06b44469a5e430374f2"/>
+				<rom name="0 - 0006.png" size="0x178d006" crc="be606943" sha1="e28069d2608cb1bb13a3753ba520a0254112e86b"/>
+				<rom name="0 - 0007.png" size="0x191749e" crc="8bfd1ea8" sha1="1a9b78ac4238dfa437285e42f0bc7f8fd4a45e92"/>
+				<rom name="0 - 0008.png" size="0x1756da2" crc="720920cb" sha1="9291de9a4ecbe42ad3eb0fceb03edaf3824de3e9"/>
+				<rom name="0 - 0009.png" size="0x178b5cb" crc="c1c51104" sha1="a2ff91851ebae16649a06474408bea1a2fcb9aba"/>
+				<rom name="0 - 0010.png" size="0x19850b0" crc="520272eb" sha1="5d40da3e50581201c87f5460bbef1c328675a3cc"/>
+				<rom name="0 - 0011.png" size="0x19b8863" crc="77aa6918" sha1="b2095780c92fb55926e8e26759ebc803fe78d677"/>
+				<rom name="0 - 0012.png" size="0x14630ba" crc="d4a4f360" sha1="af125bddb75aba6ffca1a68dd6f226a15cbd7ce6"/>
```

Comment:
I have no problem with a single space before the `/` in a self-closing element like `<tag attr="val" />` – it’s the spaces in the Japanese titles I object to.  Japanese is written without spaces.

Reply:
Yes, but on this XML there was sometimes no spaces, others one and others two. I've nnormalized it to no spaces